src/ert/job_queue/queue_differ.py

```diff
@@ -17,6 +17,18 @@ def add_state(self, queue_index: int, iens: int, state: JobStatus) -> None:
         self._qindex_to_iens[queue_index] = iens
         self._state.append(state)
 
+    def get_old_and_new_state(
```

Comment:
Was wondering (as I don't like the name of this function too much) to have just a single function called:
```python
def get_state():
     return copy.copy(self._state)
```
while `new_state = [job.status.value for job in job_list]` ?